test/framework/functional/framework.go

```diff
@@ -414,27 +408,27 @@ func (f *CollectorFunctionalFramework) addOutputContainers(b *runtime.PodBuilder
 	log.V(2).Info("Adding outputs", "outputs", outputs)
 	for _, output := range outputs {
 		switch output.Type {
-		case logging.OutputTypeSyslog:
+		case obs.OutputTypeSyslog:
 			if err := f.AddSyslogOutput(b, output); err != nil {
 				return err
 			}
-		case logging.OutputTypeKafka:
+		case obs.OutputTypeKafka:
 			if err := f.AddKafkaOutput(b, output); err != nil {
 				return err
 			}
-		case logging.OutputTypeElasticsearch:
+		case obs.OutputTypeElasticsearch:
 			if err := f.AddES7Output(b, output); err != nil {
 				return err
 			}
-		case logging.OutputTypeHttp:
+		case obs.OutputTypeHTTP:
 			if err := f.AddVectorHttpOutput(b, output); err != nil {
 				return err
 			}
-		case logging.OutputTypeSplunk:
+		case obs.OutputTypeSplunk:
 			if err := f.AddSplunkOutput(b, output); err != nil {
```

Comment:
The second parameter to `AddSplunkOutput` is unused and could be removed.

Reply:
same comment here